### DISCUSSION TITLE
Add `postgres` rules for CodeQL

### DISCUSSION THREAD

**Discussion by Minigugus** [porsager/postgres#193] [Ideas]
[CodeQL](https://semmle.com/codeql) is an advanced code analysis tool, which is now [part of Github](https://securitylab.github.com/tools/codeql).

[`pg` is already supported by CodeQL](https://help.semmle.com/QL/learn-ql/javascript/introduce-libraries-js.html#databases), maybe `postgres` could also [have its own rules](https://github.com/github/codeql/blob/master/CONTRIBUTING.md#submitting-a-new-experimental-query) ?

**Comment by porsager** [comment]
Oh yeah, that's really interesting! 

I suppose that would require a PR to the CodeQL repo, so if anyone feels up for the task, maybe mention this issue in the PR ? 🙂
